homework21/tests/conftest.py

```diff
@@ -0,0 +1,101 @@
+"""Contest for Homework 21. Task : bank deposit, task : library."""
+import pytest
+
+from homework21.source.homework11_bank_deposit import Bank, Deposit
+from homework21.source.homework11_library import Book, User
+from logging_module import setup_logging
+
+
+logger = setup_logging()
+
+
+@pytest.fixture(scope="function")
```

Comment:
function scope by default , so you do not need to write it everytime
```suggestion
@pytest.fixture()
```

Reply:
Done!